lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_priority.rb

```diff
@@ -11,6 +11,12 @@ class ContentBreadcrumbsBasedOnPriority
         transition_period: "d6c2de5d-ef90-45d1-82d4-5f2438369eea",
       }.freeze
 
+      BREXIT_TAXONS = {
+        brexit: "d6c2de5d-ef90-45d1-82d4-5f2438369eea",
+        brexit_business: "634fd193-8039-4a70-a059-919c34ff4bfc",
+        brexit_citizen: "614b2e65-56ac-4f8d-bb9c-d1a14167ba25",
```

Comment:
If you think it makes more sense for us to wait with this, @hannako, I'm happy to convert it into a draft and do the rework on this PR.

Reply:
I think we want the tracking to be live as soon as the child taxon super breadcrumbs are added to the pages. So you could push the commit to my branch. Or to keep it simple, if we merge your work first, when my PR to add the breadcrumbs is deployed it will have the correct tracking.